packages/webdriver-utils/package.json

```diff
@@ -7,6 +7,10 @@
     "url": "https://github.com/percy/cli",
     "directory": "packages/webdriver-utils"
   },
+  "publishConfig": {
+    "access": "public",
+    "tag": "latest"
```

Comment:
lets change all the `tag` to `alpha` ( even though we'll specifiy `--dist-tag` while publishing, this is for sanity)